frontends/smv_encoder.h

```diff
@@ -59,9 +61,9 @@ class SMVEncoder
   std::unordered_map<std::string, smt::Term>  unsignedbv_;
   std::deque<std::pair<int, smt::Term>> transterm_;
   std::vector<int> caselist_;
-  std::unordered_map<int, smt::Term> casecheck_;
-  std::unordered_map<int, smt::Term> casestore_;
-  std::vector<std::pair<smt::Term, smt::Term>> caseterm_;
+  std::vector<smt::Term> casecheck_;
```

Comment:
Could you add short comments for these three vectors please? And actually the more comments the better for these data structures. You can use this comment format: `///< describe what this data structure is used for`. Example:
```
///< vector of booleans, each element is an Or of all the conditions in a case statement.
///< used to check that the conditions cover all possibilities (required by nuXmv manual)
```